### ISSUE TITLE
Opens media player on WebBLE (iOS)

### ISSUE THREAD

**Issue by christophkamp** [AshleyF/briefcubing#3]
Hi there,

curious issue. When clicking on any link in the menu or Buttons in the App WebBLE opens an empty media player window. This only happens when a cube is connected.

Thanks

Christoph

**Comment by AshleyF** [comment]
Yes, this is a bug. The [NoSleep.js](https://github.com/richtr/NoSleep.js) library being used works by playing a fake bit a media in the background; preventing the device from sleeping. This is very useful when drilling algs while navigating with the Giiker cube rather than touching the screen (U for Next, U' for Repeat).

However, the WebBLE browser doesn't properly support playing *background* media and shows the player.

As a workaround, there is a setting at briefcubing.com (Allow App Sleep) that can disable this.

I'll look into a proper fix...
